doc/nrf/releases_and_maturity/migration/migration_guide_1.x_to_2.x.rst

```diff
@@ -1,3 +1,4 @@
+:orphan:
```

Comment:
Shouldn't there be space after this line? Applies to all the files that are orphaned.